crates/wasi-component-loader/src/lib.rs

```diff
@@ -127,6 +128,7 @@ impl ComponentLoader {
                 headers::map(&mut types)?;
                 context::map(&mut types)?;
                 context::map_shared(&mut types)?;
+                http_client::map(&mut types)?;
```

Comment:
IMO `inject_mapping` or something like this would be clearer than `map`. When I read `map` here I understand something like `Iterator::map` / `Operation::map`, not "mapping"